src/redecomp/transfer/MatrixTransfer.cpp

```diff
@@ -385,8 +384,7 @@ MPIArray<int> MatrixTransfer::buildRecvTrialElemDofs( const RedecompMesh& trial_
           recv_trial_elem_offsets[src_trial_r] = std::move( send_vals );
         } );
     // send parent trial vdofs back to test redecomp rank
-    auto dof_offsets = parent_trial_fes_.GetDofOffsets();
-    auto first_dof = HYPRE_AssumedPartitionCheck() ? dof_offsets[0] : dof_offsets[rank];
+    auto first_dof = parent_trial_fes_.GetMyDofOffset();
```

Comment:
Does this do the same thing as what the two removed lines were doing? Or was this a bug?